R/graded.R

```diff
@@ -1,27 +1,123 @@
-#' Graded object for submission value
+#' Signal a final grade for a student's submission
 #'
-#' The return value from `graded` should be returned by every
-#' `*-check` chunk when used with [grade_learnr()].
+#' `graded()` is used to signal a final grade for a submission. In general, it
+#' is most helpful when used in [grade_this()], but it is used internally by
+#' other grading functions to signal that the student's submission has been
+#' graded. If you're using [grade_this()], you'll most likely want to use the
+#' helper functions `pass()`, `fail()`, `pass_if_equal()`, and
+#' `fail_if_equal()`, rather than to call `graded()` directly.
 #'
-#' `graded()` objects are signaled to the calling functions.
+#' @section Usage in `grade_learnr()`: 
```

Comment:
In a future PR, we should consider renaming `grade_learnr()` to something away from `grade_*()` as the usage of `grade_learnr()` is NOT the same as `grade_*()` functions

Reply:
We could still export it, but it should be soft deprecated and wrap around the newly named function

Reply:
Since the `tutorial_option()` is `exercise.checker`, how about `gradethis_exercise_checker()`?

Reply:
Works for me!!

Reply:
Tracking in new issue #204